.github/workflows/deploy.yml

```diff
@@ -0,0 +1,59 @@
+name: Test and Deploy
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+    env:
+        DEBUG: True
+        SECRET_KEY: changeme
+        DATABASE_URL: postgres://postgres@127.0.0.1/dienst2?atomic_requests=True
+        CACHE_URL: dummycache://
+        SEARCH_URL: simple://
+        OIDC_RP_CLIENT_ID: changeme
+        OIDC_RP_CLIENT_SECRET: changeme
+
+    services:
+      postgres:
+        image: postgres:9.6
+        env: 
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: dienst2
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install prerequisites
+      run: | 
+        sudo apt-get install libpq-dev
+        sudo apt-get install libsasl2-dev python-dev libldap2-dev libssl-dev
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Run migrations
+      run: python manage.py migrate
+    - name: Run tests
+      run: python manage.py test
+    - name: Deploy to Quay
+      uses: docker/build-push-action@v2
+      with:
+        username: ${{ secrets.QUAY_USERNAME }}
```

Comment:
The build-push-action@v2 action requires different configuration than v1 does, see and upgrade notes at https://github.com/docker/build-push-action/blob/master/UPGRADE.md and e.g. https://github.com/WISVCH/docker-spring-boot-base/commit/596a2c78ae3222d8a6427391feeff7550db31080.

Reply:
Made changes based on the provided links. Let me know if there is still any issues with this

Reply:
Looks good!